addons/binding/org.openhab.binding.zoneminder/src/main/java/org/openhab/binding/zoneminder/handler/ZoneMinderBaseThingHandler.java

```diff
@@ -367,7 +367,7 @@ protected State getChannelStringAsStringState(String channelValue) {
             }
 
         } catch (Exception ex) {
-            logger.error(ex.getMessage());
+            logger.error("", ex.getMessage());
```

Comment:
I would log the exception instead, it is an error, I want to be able to debug the stacktrace